DataReport/index.js

```diff
@@ -0,0 +1,38 @@
+const { Logger } = require('werelogs');
+const createMongoParams = require('../utils/createMongoParams');
+const { collectBucketMetricsAndUpdateBucketCapacityInfo } = require('./collectBucketMetricsAndUpdateBucketCapacityInfo');
+const S3UtilsMongoClient = require('../utils/S3UtilsMongoClient');
+
+const log = new Logger('s3utils:DataReport');
+
+function main() {
+    const mongoClient = new S3UtilsMongoClient(createMongoParams(log));
+    mongoClient.setup(err => {
+        if (err) {
+            log.error('error connecting to mongodb', {
+                error: err,
+            });
+            process.exit(1);
+        }
+        collectBucketMetricsAndUpdateBucketCapacityInfo(mongoClient, log, err => {
+            if (err) {
+                log.error('error collecting bucket metrics and updating bucket capacity info', {
+                    error: err,
+                });
+                process.exit(1);
+            }
```

Comment:
Just for consistency, we can call `stop()` here, to also make sure we control the return code.